src/js/config/config.template.js

```diff
@@ -0,0 +1,6 @@
+var configDev = {
+  // Defines the Marathon API URL,
+  // leave empty to use the same as the UI is served.
+  apiURL: ""
```

Comment:
I don't think we need to expose the other properties for now. Let me know if you think otherwise, though.